packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.stories.ts

```diff
@@ -0,0 +1,100 @@
+import { ONYX_COLORS, type OnyxIconProps } from "@/index";
+import happyIcon from "@sit-onyx/icons/emoji-happy-2.svg?raw";
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryContext, StoryObj } from "@storybook/vue3";
+import OnyxIcon from "./OnyxIcon.vue";
+import { ICON_SIZES } from "./types";
+
+const ALL_ICONS = import.meta.glob("../../../node_modules/@sit-onyx/icons/src/assets/*.svg", {
+  as: "raw",
+  eager: true,
+});
+
+/**
+ * Mapping between icon SVG content (key) and icon name (value).
+ * Needed to display a labelled dropdown list of all available icons.
+ */
+const iconLabels = Object.entries(ALL_ICONS).reduce<Record<string, string>>(
+  (labels, [filePath, content]) => {
+    labels[content] = filePath.split("/").at(-1)!.replace(".svg", "");
+    return labels;
+  },
+  {},
+);
+
+/**
+ * Component to display icons. Supports all inline SVG icon libraries.
+ * We recommend using the official icons from `@sit-onyx/icons`.
+ *
+ * When importing icons, make sure to add `?raw` after the file name as shown in the examples to import the SVG content
```

Comment:
does this apply to all icons, even from other libs? or rather
```suggestion
 * When importing icon files, make sure to add `?raw` after the file name as shown in the examples to import the SVG content
```
or
```suggestion
 * When importing onyx icons, make sure to add `?raw` after the file name as shown in the examples to import the SVG content
```

Reply:
Its needed whenever a `.svg` file is directly imported so also for other icon libraries.